### PR TITLE
www.bulurum.com

### PR DESCRIPTION
**Pull request by gork7777**
issue: #88581

### CODE REVIEW

**Comment by Sergey-Lyapin** [comment]
please also check the main page and some random pages
bulurum

**Comment by Sergey-Lyapin** [comment]
remove this rule:
`bulurum.com##a[href*="advertising.bulurum.com"][target="_blank"]`
and add:
```
@@||bulurum.com/v*/Images/Mvc/home/advertise_*.png
bulurum.com##div[id^="Banner"]
bulurum.com##.MobileContainer
```

File: TurkishFilter/sections/specific.txt [hunk 1]
```diff
@@ -45,6 +45,11 @@ yemektarifleri.org,lifebursa.com,mshowto.org,macizlee1.com,macizlee.ne,dunyaatla
 ! ###adStarter
 fullhdfilmcibaba.com,720pizlemek.com,1080pfilmizle.net,fullhdfilmcidayi.com,hdfilmizledayi.com,720-pizleme.com,fullhdfilmizlebaba.com###adStarter
 !
+@@||bulurum.com/v*/Images/Mvc/home/advertise_*.png
```
Comment (zloyden) [inline TurkishFilter/sections/specific.txt:48]: Move this to the Turkish - whitelist.txt.

File: TurkishFilter/sections/specific.txt [hunk 2]
```diff
@@ -45,6 +45,11 @@ yemektarifleri.org,lifebursa.com,mshowto.org,macizlee1.com,macizlee.ne,dunyaatla
...
+bulurum.com##div[id^="Banner"]
+bulurum.com##.MobileContainer
```
Comment (zloyden) [inline TurkishFilter/sections/specific.txt:50]: Move `bulurum.com##.MobileContainer` to the Annoyances - mobile-app.txt.

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**